soh/src/overlays/actors/ovl_En_Wallmas/z_en_wallmas.c

```diff
@@ -417,6 +431,13 @@ void EnWallmas_Die(EnWallmas* this, PlayState* play) {
         Item_DropCollectibleRandom(play, &this->actor, &this->actor.world.pos, 0xC0);
         Actor_Kill(&this->actor);
     }
+    if (this->actor.params == WMT_SHADOWTAG) {
+        if (!CVarGetInteger("gShadowTag", 0)) {
+            Actor_Kill(&this->actor);
```

Comment:
so is the idea here that after turning off shadow tag you need to hit the wallmaster once (or wait for it to return to the ceiling) before it'll go away?